### ISSUE TITLE
 dashboard-page-separator  seems to be broken ? 

### ISSUE THREAD

**Issue by drishal** [emacs-dashboard/emacs-dashboard#507]
on setting `  (setq dashboard-page-separator  "\n\f\n")` the alignment of the lines seems to be broken and now looks like this 
![image](https://github.com/emacs-dashboard/emacs-dashboard/assets/32875739/453b8318-79ea-4fd4-923b-668b039adac6)

whereas previously it used  to look like this 
![image](https://github.com/emacs-dashboard/emacs-dashboard/assets/32875739/ed1975ea-be20-4cfa-96c9-d509772082ac)

Can this issue please be fixed?

My current dashboard config
```
(use-package dashboard
  :demand t
  :config
  (add-hook 'elpaca-after-init-hook #'dashboard-insert-startupify-lists)
  (add-hook 'elpaca-after-init-hook #'dashboard-initialize)
  (setq dashboard-icon-type 'all-the-icons)
  (setq dashboard-heading-icons '((recents   . "file-text")
                                  (bookmarks . "book")
                                  (agenda    . "calendar")
                                  (projects  . "rocket")
                                  (registers . "database")))
  (setq dashboard-set-heading-icons t)
  ;; Content is not centered by default. To center, set
  (setq dashboard-set-file-icons t)
  (setq dashboard-banner-logo-title "Emacs Is More Than A Text Editor!")
  (setq dashboard-startup-banner `logo) ;; use standard emacs logo as banner
  (setq dashboard-center-content t)
  (setq dashboard-set-navigator t)
  (setq dashboard-page-separator  "\n\f\n")
  (setq dashboard-items '((recents . 5)
                          (bookmarks . 5)
                          (projects . 3)
                          ))
  (dashboard-setup-startup-hook)
  (setq initial-buffer-choice (lambda () (get-buffer-create "*dashboard*")))
  )
```

**Comment by jcs090218** [comment]
I'm on it.

**Comment by jcs090218** [comment]
Should be fixed in c7ae464c5dd54615fcb7806bd5a22aa9f8e1ce00.

**Comment by drishal** [comment]
> Should be fixed in [c7ae464](https://github.com/emacs-dashboard/emacs-dashboard/commit/c7ae464c5dd54615fcb7806bd5a22aa9f8e1ce00).

Thanks, fixed
